force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
### Testing Best Practice: Uncommented assertion
#### Severity
Warning

#### Finding
This assertion does not include any comments. Please add a comment that describes rationale of the assertion.

#### Why is this a problem?
Assertions let you test the behaviour of your application, increasing your confidence that your logic works as expected. Adding precise assertions to your tests is one of the most effective ways to detect and correct bugs. Using comments to describe the rationale of each assertion has the added benefit of documenting the inner working of the application logic, enhancing maintainability.

#### Help and documentation

* [Inaccurate Comments (CWE)](https://cwe.mitre.org/data/definitions/1116.html)